### ISSUE TITLE
tarfile_calibration does not visualize rectified images after calibration

### ISSUE THREAD

**Issue by trigal** [ros-perception/image_pipeline#285]
in tarfile_calibration.py there is a waitKey define with 6 ms as delay (as reported in [opencv documentation](http://docs.opencv.org/2.4/modules/highgui/doc/user_interface.html#waitkey) . 
This doesn't make sense since waitkey() should wait for a pressed key, then releasing the while inside the display() define. 

```python
def waitkey():
    k = cv2.waitKey(6)
    return k

def display(win_name, img):
    cv2.namedWindow(win_name, cv2.WINDOW_NORMAL)
    cv2.imshow( win_name,  numpy.asarray( img[:,:] ))
    k=-1
    while k ==-1:
        k=waitkey()
    cv2.destroyWindow(win_name)
    if k in [27, ord('q')]:
        rospy.signal_shutdown('Quit')
```

this should be the desired behavior: 
```python
def waitkey():
    k = cv2.waitKey(0)
    return k
```

**Comment by k-okada** [comment]
I have rewrite #286  and post new PR #340, but not sure how this related to the "does not visualize rectified images after calibration" can you provide sample tar file and instructions to how to reproduce this problem?

**Comment by JWhitleyWork** [comment]
Closing stale issue which appears to also be fixed by #340. Can re-open if OP replies or offers feedback.
